fac-graduate-programme/Nova-Foundation
# Feature/service page

#28 

## Changes:
1. Edit `clip-path` attribute in the Logo component to `clipPath` since it throws a warning in the console.
2. Create the home page with function for redirection

## How to test?
Your app should like this:

```javascript
import React from "react";
import { BrowserRouter as Router, Route } from 'react-router-dom'

import "./App.css";
import Home from './Component/Pages/Home'
import Footer from './Component/CommonComponent/Footer'


function App() {
  return (
    <div className="App">
      <Router>
        <Route path='/home' component={Home} />
      </Router>
      <Footer />

    </div>
  );
}

export default App;
```
3. Delete home h3 from header. 
It was quite hard to for me to style the title and get the header as in the Figma, so I moved it inside the page component to have more control.
